.github/workflows/test.yml

```diff
@@ -32,7 +32,7 @@ jobs:
         submodules: true
 
     - name: Install dependencies
-      run: sage --pip install pycryptodomex
+      run: sage -pip install git+https://github.com/cfrg/draft-irtf-cfrg-vdaf@72227878#subdirectory=poc
```

Comment:
Curiously. What is `@72227878` for? Doesn't look like branch name or commit.

Reply:
It's a truncated commit, it refers to a commit in the draft-irtf-cfrg-vdaf repo. We'll eventually replace this with the next tag, which will be draft-irtf-cfrg-vdaf-11.

Reply:
Updated to fefc5a2f563e70e73a74d3c57b7145e0dd1c2e3f.